### PR TITLE
Sandwich corrected

### PR DESCRIPTION
**Pull request by AitorBengoechea**

### CODE REVIEW

**Comment by AitorBengoechea** [comment]
I have decided that if after applying the `sandwich` method, what is returned is a scalar, that it should not be part of `CategoryCov`, since no method inside the class is useful for this output.

File: sandy/core/cov.py [hunk 1]
```diff
@@ -1468,19 +1468,7 @@ def sandwich(self, s, rows=None, threshold=None):
...
-
-        >>> index = [1, 2, 3]
-        >>> c = pd.DataFrame(np.diag(np.array([1, 2, 3])), index=index, columns=index)
-        >>> s = pd.Series([1, 2, 3], index=pd.Index([1, 2, 3]))
-        >>> sandy.CategoryCov(c).sandwich(s)
-        	          1	          2	          3
-        1	1.00000e+00	0.00000e+00	0.00000e+00
-        2	0.00000e+00	8.00000e+00	0.00000e+00
-        3	0.00000e+00	0.00000e+00	2.70000e+01
+        36.0
```
Comment (luca-fiorito-11) [inline sandy/core/cov.py:1471]: You should update the `Returns` section in the docstring

File: sandy/core/cov.py [hunk 2]
```diff
@@ -1468,19 +1468,7 @@ def sandwich(self, s, rows=None, threshold=None):
...
 
         >>> S = np.array([1, 2, 3])
         >>> var = pd.Series([1, 2, 3])
```
Comment (luca-fiorito-11) [inline sandy/core/cov.py:1474]: please call the sensitivity `s` and the variance `var`. Also in the other tests

File: sandy/core/cov.py [hunk 3]
```diff
@@ -1493,16 +1481,15 @@ def sandwich(self, s, rows=None, threshold=None):
         2	0.00000e+00	0.00000e+00	2.70000e+01
         """
         if pd.DataFrame(s).shape[1] == 1:
-            s_ = sandy.CategoryCov.from_var(s).data
-            sandwich = self._gls_Vy_calc(s_, rows=rows)
+            s_ = pd.Series(s)
```
Comment (luca-fiorito-11) [inline sandy/core/cov.py:1484]: add a comment here that this will return a scalar

File: sandy/core/cov.py [hunk 4]
```diff
@@ -1493,16 +1481,15 @@ def sandwich(self, s, rows=None, threshold=None):
...
         if pd.DataFrame(s).shape[1] == 1:
-            s_ = sandy.CategoryCov.from_var(s).data
-            sandwich = self._gls_Vy_calc(s_, rows=rows)
+            s_ = pd.Series(s)
+            sandwich = s_.dot(self.data.dot(s_.T))
+            return sandwich
         else:
             s_ = pd.DataFrame(s).T
             sandwich = self._gls_Vy_calc(s_, rows=rows)
         if threshold is not None:
```
Comment (luca-fiorito-11) [inline sandy/core/cov.py:1490]: then all this part should be in `else`

**Review by luca-fiorito-11 — CHANGES_REQUESTED** [review]

**Review by luca-fiorito-11 — APPROVED** [review]

**Pull request merged**